src/awkward/_nplikes/numpylike.py

```diff
@@ -16,7 +16,7 @@
     overload,
 )
 
-ShapeItem: TypeAlias = "SupportsInt | None"
```

Comment:
@henryiii do you know which ruff rule is wiping out these quotes?

Reply:
Yes, update Ruff to v0.0.243, it was noted in https://github.com/charliermarsh/ruff/issues/2613 and fixed in https://github.com/charliermarsh/ruff/pull/2614 and released.